src/UglyToad.PdfPig/Content/Line.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using UglyToad.PdfPig.Geometry;
+using System.Linq;
+
+namespace UglyToad.PdfPig.Content
+{
+    /// <summary>
+    /// A line.
+    /// </summary>
+    public class Line
+    {
+        /// <summary>
+        /// The text of the line.
+        /// </summary>
+        public string Text { get; }
+
+        /// <summary>
+        /// The text direction of the line.
+        /// </summary>
+        public TextDirection TextDirection { get; }
+
+        /// <summary>
+        /// The rectangle completely containing the line.
+        /// </summary>
+        public PdfRectangle BoundingBox { get; }
+
+        /// <summary>
+        /// The words contained in the line.
+        /// </summary>
+        public IReadOnlyList<Word> Words { get; }
+
+        /// <summary>
+        /// Create a new <see cref="Word"/>.
+        /// </summary>
+        /// <param name="words">The letters contained in the word.</param>
+        public Line(IReadOnlyList<Word> words)
+        {
+            if (words == null)
+            {
+                throw new ArgumentNullException(nameof(words));
+            }
+
+            if (words.Count == 0)
+            {
+                throw new ArgumentException("Empty words provided.", nameof(words));
+            }
+
+            Words = words;
+
+            Text = string.Join(" ", words.Where(s => !string.IsNullOrWhiteSpace(s.Text)).Select(x => x.Text));
+
+            var minX = words.Min(x => x.BoundingBox.Left);
+            var minY = words.Min(x => x.BoundingBox.Bottom);
+            var maxX = words.Max(x => x.BoundingBox.Right);
+            var maxY = words.Max(x => x.BoundingBox.Top);
+            BoundingBox = new PdfRectangle(minX, minY, maxX, maxY);
+
+            TextDirection = words[0].TextDirection;
```

Comment:
Do we need some additional guarantees here? I'm not sure if the user might expect text direction of all words in a line to be consistent, I'm trying to think of both possible use cases but it might be better to check `words.All(x => x.TextDirection == words[0].TextDirection)` or whatever the nicer way of expressing that is, I'll defer to you on this since you're working with the API.

Reply:
Good point, will use `TextDirection = TextDirection.Unknown` if several directions are present.